.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+---
+version: 2.1
+
+orbs:
+  slack: circleci/slack@4.1.3
+
+executors:
+  python36:
```

Comment:
any reason why we shouldnt use a new python version? 3.6 is almost considered old in some communities at this point

Reply:
Thanks for pointing out, I'll update the version we use in CI to 3.9.
My original intention was to "test" with the oldest version we support, because the library should be supported from 3.6 onwards as we currently don't have good testing framework for different python versions, but I guess its not as meaningful and we should invest some time on having proper tests for the different versions.

Reply:
There are ways of testing multiple versions with a bit of work. Might be a tiny bit more complicated with `poetry` but it would roughly equivalent to this:

https://github.com/MichaelAquilina/flake8-spellcheck/blob/master/.circleci/config.yml#L46-L64

Reply:
yep thats what i was referring too, but I think we can leave it for a followup PR